README.md

```diff
@@ -109,6 +109,12 @@ To run tests, run the following command accordingly:
 - Twitter: [@twitterhandle](https://twitter.com/@monzo200)
 - LinkedIn: [LinkedIn](https://https://www.linkedin.com/in/felixnyamai/)
 
+👤 **Ajayi Ridwan**
+
+- GitHub: [@githubhandle](https://github.com/Ridwanullahi-code)
+- Twitter: [@twitterhandle](https://twitter.com/Ridwanullahi22)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ajayi-ridwan-2a1515199/)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```

Comment:
- [ ] It would be nice if you setup linters in your repository because linters helps and prevents you from making those errors in your code which can break your app entirely 😨